app/controllers/login.js

```diff
@@ -1,5 +1,7 @@
-import Ember from 'ember';
+import { Controller, computed } from 'ember';
 
 import OsfTokenLoginControllerMixin from 'ember-osf/mixins/osf-token-login-controller';
 
-export default Ember.Controller.extend(OsfTokenLoginControllerMixin);
+export default Controller.extend(OsfTokenLoginControllerMixin, {
+    login: computed.alias('model'),
```

Comment:
Thanks. I really appreciate your willingness to experiment a little as we try out and refine the new linting rules.